template/module/wizards/wizard_model_view.xml

```diff
@@ -5,7 +5,7 @@
 <openerp>
 <data>
 
-<record id="wizard_model_view" model="ir.ui.view">
+<record id="wizard_model_view_form" model="ir.ui.view">
```

Comment:
@pedrobaeza - the model in this instance is `module.wizard_model`.

I think we should maybe revisit our wizard naming, although I dunno if it makes sense to do it here